src/VisualStudio/Core/Def/Implementation/AbstractEditorFactory.cs

```diff
@@ -29,18 +28,13 @@ internal abstract partial class AbstractEditorFactory : IVsEditorFactory, IVsEdi
         private readonly Package _package;
         private readonly IComponentModel _componentModel;
         private Microsoft.VisualStudio.OLE.Interop.IServiceProvider _oleServiceProvider;
-        private readonly IVsEditorAdaptersFactoryService _editorAdaptersFactoryService;
-        private readonly IContentTypeRegistryService _contentTypeRegistryService;
         private readonly IWaitIndicator _waitIndicator;
         private bool _encoding;
 
         protected AbstractEditorFactory(Package package)
         {
             _package = package ?? throw new ArgumentNullException(nameof(package));
             _componentModel = (IComponentModel)ServiceProvider.GetService(typeof(SComponentModel));
-
-            _editorAdaptersFactoryService = _componentModel.GetService<IVsEditorAdaptersFactoryService>();
-            _contentTypeRegistryService = _componentModel.GetService<IContentTypeRegistryService>();
             _waitIndicator = _componentModel.GetService<IWaitIndicator>();
```

Comment:
❗️ This field should be lazily initialized as well per the original bug report (avoid unnecessary calls to `GetService` in this constructor).